freezeyt/cli.py

```diff
@@ -9,8 +9,10 @@
 
 
 @click.command()
-@click.argument('module_name')
-@click.argument('dest_path', required=False)
+@click.argument('module_name', required=False)
```

Comment:
I dunno. we use `module_name` for two different things, there can be *module* or *app*. Both of them will be work but create name for something what is doing two things is difficult. I can name it as *app* because that is what we want at the end to finish the task but it is misleading because module can be inserted as well.
If you agree, I suggest to rewrite readme where we should mention that the right way how to use cli is to specify *app*. If you specify *module* then freezeyt handle it but it is not primary intention of cli. Now I feel the readme is written the other way around.

Reply:
Finally, I kept `module_name`, it make sense to put just module when I realized that most of application will use variable with name `app`.

`import_variable_from_module` is still relevant question.

Reply:
I'd actually go with `app` for the CLI  option: that's the thing the user is telling freezeyt about. You shorten the input to just the module name, but that's still specifying which app to freeze.
(Compare to `--import-config`: we could easily add a default variable name for that option, and even if people specify the config's *module*, they'd do it to tell freezeyt about the config to use.)

About `import_variable_from_module`: that's an internal function, so its name should help freezeyt developers. Ths CLI/config options are for *users*, who shouldn't care what happens inside freezeyt.
Inside freezeyt, it might be more clear to use `app` only for the actual WSGI callable, and `app_specification` for the `module_name`/`module_name:var_name` string.